default.nix

```diff
@@ -7,10 +8,11 @@ let
         niv = (import sources.niv { }).niv;
         # include local sources in your Nix projects, while taking gitignore files into account
         # https://github.com/hercules-ci/gitignore.nix
-        gitignoreSource = (import sources.gitignore { }).gitignoreSource;
+        gitignoreSource = (import sources.gitignore { lib=pkgs.lib; }).gitignoreSource;
```

Comment:
I fixed this already; sorry for the rebase

Reply:
Thank you for your quick reply :)

I rebased it.

About testing:
Till now I never used github actions, but always circleci instead. I will try to figure out how to add the test.